apis/cloud.redhat.com/v1alpha1/clowdapp_webhook.go

```diff
@@ -35,6 +35,11 @@ func (r *ClowdApp) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
can we remove this comment? I think it's pretty obvious that we should edit this file.